enhancements/network/whereabouts-sticky-ips.md

```diff
@@ -0,0 +1,84 @@
+---
+title: whereabouts-sticky-ips
+authors:
+  - "@dougbtv"
+reviewers:
+  - "@s1061123"
+  - "@zshi-redhat"
+approvers:
+  - "@s1061123"
+  - "@zshi-redhat"
+creation-date: 2020-08-05
+last-updated: 2020-08-05
+status: provisional
+---
+
+
+# Whereabouts "Sticky IP addresses"
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This feature implements functionality for Whereabouts where workloads are assigned an IP address, and when those workloads are terminated and restarted, are assigned the same IP address.
+
+## Motivation
+
+This functionality has been requested to provide reliable IP addresses for workloads that advertise their IP address in a method other than Kubernetes services, and where external components require that those same IP addresses be reachable regardless if the workload has been restarted.
```

Comment:
What does "advertise their IP address in a method other than Kubernetes services" mean?